Capstone Project/ViewControllers/Workouts/CurrentWorkoutViewController.m

```diff
@@ -232,6 +246,13 @@ - (void)tableView:(UITableView *)tableView commitEditingStyle:(UITableViewCellEd
     }
 }
 
+-(void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath{
+    if (tableView == self.autocompleteTableView){
+        tableView.hidden = YES;
+        self.exerciseTextField.text = [self.autocompleteExercises objectAtIndex:indexPath.row];
+    }
+}
+
 
```

Comment:
The add button is always enabled. It should be enabled only when there are all required entries with valid data